fsharp/oxpecker/Program.fs

```diff
@@ -0,0 +1,41 @@
+open System.Threading.Tasks
+open Microsoft.AspNetCore.Builder
+open Microsoft.AspNetCore.Hosting
+open Microsoft.Extensions.DependencyInjection
+open Microsoft.Extensions.Hosting
+open Microsoft.Extensions.Logging
+open Oxpecker
+
+let emptyHandler: EndpointHandler = fun _ -> Task.CompletedTask
+
+let userIdHandler: EndpointHandler =
+    fun ctx ->
+        match ctx.TryGetRouteValue<string>("id") with
+        | Some id -> ctx.WriteText(id)
+        | None -> Task.CompletedTask
+
+let endpoints =
+    [ GET [ route "/" emptyHandler; route "/user/{id?}" userIdHandler ]
```

Comment:
Two comments
1. I think id is not optional in the benchmark so it can be rewritten to `route "/user/{id}" userIdHandler`
2. To make it overall endpoints list look nicer I'd rewrite it into
```fsharp
let endpoints = [ 
    GET [ 
        route "/" emptyHandler
        route "/user/{id}" userIdHandler 
    ]      
    POST [ 
        route "/user" emptyHandler 
    ] 
]
```

Reply:
I can tackle 1. 2. is a bit more challenging with fantomas installed. :)

Reply:
Thank you, approved!